manifests/repos.pp

```diff
@@ -43,12 +44,12 @@
         }
 
         apt::source { 'grafana':
-          location => 'https://packagecloud.io/grafana/stable/debian/',
-          release  => 'jessie',
+          location => 'https://packages.grafana.com/oss/deb',
+          release  => 'stable',
           repos    => 'main',
           key      =>  {
-            'id'     => '418A7F2FB0E1E6E7EABF6FE8C2E73424D59097AB',
-            'source' => 'https://packagecloud.io/gpg.key',
+            'id'     => '8C8C34C524098CB6',
```

Comment:
Thats the old key id. Here's how I determined the new id:

```
╔ ☕️  gene:~
╚ᐅ curl https://packages.grafana.com/gpg.key |gpg --with-colons |awk -F: '/^pub:/ { print $5 }'
gpg: WARNING: no command supplied.  Trying to guess what you mean ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1694  100  1694    0     0   9163      0 --:--:-- --:--:-- --:--:--  9156
8C8C34C524098CB6
```

Using a fresh docker container I can verify that the above is the correct key:

```
root@e3c649e983e0:/# cat /etc/apt/sources.list.d/grafana.list
deb https://packages.grafana.com/oss/deb stable main

root@e3c649e983e0:/# apt-get update
Hit:1 http://security.ubuntu.com/ubuntu xenial-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu xenial InRelease
Hit:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
Get:4 https://packages.grafana.com/oss/deb stable InRelease [10.4 kB]
Hit:5 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
Ign:4 https://packages.grafana.com/oss/deb stable InRelease
Get:6 https://packages.grafana.com/oss/deb stable/main amd64 Packages [9379 B]
Fetched 19.8 kB in 0s (35.9 kB/s)
Reading package lists... Done
W: GPG error: https://packages.grafana.com/oss/deb stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 8C8C34C524098CB6
W: The repository 'https://packages.grafana.com/oss/deb stable InRelease' is not signed.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Reply:
Looks like they updated their key. The `8C8C34C524098CB6` key is what is live at https://packages.grafana.com/gpg.key which is what they say to use in the install docs at http://docs.grafana.org/installation/debian/

Reply:
ah ok, what I'm seeing is a non-fatal error from the apt module:
https://forge.puppet.com/puppetlabs/apt#add-gpg-keys
There's a note there about short keys vs full fingerprints

Reply:
@genebean - Does it make sense to use the longer key since that's preferred by puppetlabs-apt?
I had tested this in #31 and it looked ok to me.

Reply:
I also used 4E40DDF6D76E284A4A6780E48C8C34C524098CB6 (manually, then wrote a PR with the same: #32) It seems Erik and I both didn't notice this PR includes a commit for the repo change, possibly because of the title of this PR. Maybe the repo change should not broken out?